mlrun/runtimes/function.py

```diff
@@ -407,4 +412,15 @@ def fake_nuclio_context(body, headers=None):
 def _fullname(project, name):
     if project:
         return '{}-{}'.format(project, name)
-    return name
\ No newline at end of file
+    return name
+
+
+def get_platform_dashboard(dashboard):
+    if '.default-tenant.' not in mlconf.dbpath:
+        return dashboard or mlconf.nuclio_dashboard
+
+    p = urlparse(mlconf.dbpath)
+    user, password, _ = refresh_credentials(p.hostname)
+    return 'https://{}:{}@{}{}'.format(
+        user, password, mlconf.nuclio_dashboard or 'nuclio-api-ex',
+        p.hostname[p.hostname.find('.'):])
```

Comment:
I don't understand what this function returns ? the nuclio api url ?
if yes, what should the value in `mlconf.nuclio_dashboard` look like ?

Reply:
@Hedingber this is the external nuclio dash url (with auth), the `mlconf.nuclio_dashboard` in this case will specify the sub-domain (xx.default-tenant...) until we fix it in 2.10